pmg/models/users.py

```diff
@@ -149,19 +149,25 @@ def gets_alerts_for(self, committee):
             committee = Committee.query.get(committee)
         return committee in self.committee_alerts
 
+    def get_followed_committee_meetings(self):
+        from ..models.resources import CommitteeMeeting
+        from ..api.schemas import CommitteeMeetingSchema
+        following = CommitteeMeeting.committee_id.in_([f.id for f in current_user.following])
+        meetings = CommitteeMeeting.query.filter(following).order_by(desc(CommitteeMeeting.date)).limit(10)
+
+        return CommitteeMeetingSchema(many=True, only=['id', 'title', 'date']).dump(meetings).data
```

Comment:
Ah, `.data` :)

You're not always going to want to return this as json, though. Just return the `meetings` object, then it's up to the caller to determine what to do with it. In the one case we can send it as-is to the template, in the other cases it needs to be JSON-ified.

Otherwise we're mixing the business logic of "find the recent meetings that this user cares about" with the view logic of "I need it in this form to show it".